lib/std/os.zig

```diff
@@ -3467,7 +3467,7 @@ pub fn waitpid(pid: pid_t, flags: u32) WaitPidResult {
     }
 }
 
-pub const Stat = if (builtin.link_libc)
+pub const Stat = if (builtin.os.tag != .wasi and builtin.link_libc)
```

Comment:
What?

Reply:
sorry I missed this I added at first for debugging. Fixed in https://github.com/ziglang/zig/pull/9227/commits/05364481e47b81ccb23799eba022adebc19f0958